features/step_definitions/work_steps.rb

```diff
@@ -709,6 +709,12 @@
   end
 end
 
+When "I set the collections of work {string} to {string}" do |work, collection|
```

Comment:
"I edit the work X to be in the collection Y"? Let's move this over to collection_steps.rb as well, next to the invite version.

Reply:
Thanks for the better words! I've updated that step, and I also added an optional `s` to collections since some features are adding more than one